astro/src/content/docs/apis/message-templates.mdx

```diff
@@ -138,7 +138,18 @@ This API does not return a JSON response body.
 
 This API is used to preview a Message Template. You pass all of the information for the Message Template in the request and a rendered version of the Message is sent back to you in the response.
 
-FusionAuth provides sample values for the `${code}`, `${verificationId}`, and `${verificationOneTimeCode}` parameters when previewing. The values for these parameters will always be `123456`, `987654`, and `abcdef` respectively. The `${code}` parameter is used to deliver a two factor code to the user while `${verificationId}` and `${verificationOneTimeCode}` are used for phone number verification messages.
+FusionAuth provides sample values for the following parameters when previewing:
+
+| Parameter                    | Sample Value                                                                 | Purpose                                                              |
+| ---------------------------- | ---------------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| `${application}`             | `{"name": "My Application", "oauthConfiguration": {"clientId": "123456"}}`   | `clientId` for themed message links                                  |
```

Comment:
For purpose, might want to mention access to the application name, too, so you can put that in your template.

Reply:
Pushed as 97bebea28a04f0e1d3b7f88965dc2739a605f39d

Reply:
See 0bbc53c0b97fd0fddfe43d2977543cb5147d790f